common/app/views/fragments/items/facia_cards/item.scala.html

```diff
@@ -17,11 +18,18 @@
 
 @defining(containerIndex == 0) { case (isFirstContainer) =>
     @item match {
+
         case content: ContentCard => {
             @contentCard(content, containerIndex, index, card.visibilityDataAttribute, isFirstContainer, isList)
         }
 
-        case paidCard: PaidCard => {
+        case paidContentOnPaidPage: PaidCard if frontProperties.exists(_.isPaidContent) && paidContentOnPaidPage.contentCard.isDefined =>{
+            @paidContentOnPaidPage.contentCard.map { c =>
+                @contentCard(c, containerIndex, index, card.visibilityDataAttribute, isFirstContainer, isList)
```

Comment:
@gidsg I have had a look around at the types. Why can't you just create a new `paidContentCardOnPaidPage` template and copy the other template with different styles or DOM layout?

Or even just change the `paidContentCard` template to take the boolean as to whether the page is a paid page or not and do logic around showing the sponsored by message around that boolean?

That means you can leave the model the way it is instead of carrying the original `ContentCard` around with you, or even have to create it in the first place.